src/frontends/tensorflow/src/utils.hpp

```diff
@@ -26,13 +26,21 @@ void set_node_name(const std::string& node_name, const std::shared_ptr<Node>& no
 bool is_conditional_edge(const std::string& input_tensor_name);
 
 template <typename T>
-void get_const_input(const NodeContext& node, int64_t input_index, std::vector<T>* vector) {
-    auto ng_input = node.get_input(static_cast<int>(input_index));
-    if (auto constant = std::dynamic_pointer_cast<opset8::Constant>(ng_input.get_node_shared_ptr())) {
+void get_const_input(const NodeContext& node, int input_index, std::vector<T>* vector) {
+    auto input_size = static_cast<int>(node.get_input_size());
+    auto node_name = node.get_name();
+    auto node_type = node.get_op_type();
+    FRONT_END_GENERAL_CHECK(0 <= input_index && input_index < input_size,
```

Comment:
do we have tensorflow frontend macro?
if not, I believe we need to create it to avoid using "[TensorFlow Frontend] Internal error: " everywhere

Reply:
I agree with you. Adjustment of the logging will be separate effort because all type of logs need to be adjusted in this case.

Reply:
Do we have a ticket or could you create a new one?